examples/gnrc_border_router/README.md

```diff
@@ -1,4 +1,144 @@
-# gnrc_networking_border_router example
+# gnrc_border_router using automatic configuration
+A new setup is possible on which only one serial interface is needed. 
+This is done through the ethos (Ethernet Over Serial) driver and UHCP.
+Ethos multiplexes serial data to separate ethernet packets from shell commands.
+UHCP is in charge of configure the wireless interface prefix and routes on the BR.
+
+The script `start_network.sh` enables a *ready-to-use* BR in only one command.
+
+## Requirements
+This functionality works only on Linux machines.
+Mac OSX support will be added in the future (lack of `tap` interface).
+
+## Setup
+First, you need to compile `ethos`.
+Go to `/dist/tools/ethos` and type:
+
+```bash
+make clean all
+```
+
+Then, you need to compile UHCP (micro Host Control Protocol).
+This tool is found in `/dist/tools/uhcp`. So, as for `ethos`:
+
+```bash
+make clean all
+```
+
+Then you can compile and flash your `gnrc_border_router` example.
+To select ethos as the serial driver, be sure that the `Makefile` has the following:
+
+```make
+ifeq (,$(filter native,$(BOARD)))
+GNRC_NETIF_NUMOF := 2
+USEMODULE += ethos gnrc_netdev2
+CFLAGS += '-DETHOS_UART=UART_DEV(0)' -DETHOS_BAUDRATE=115200 -DUSE_ETHOS_FOR_STDIO
+FEATURES_REQUIRED += periph_uart
+endif
+# include UHCP client
+USEMODULE += gnrc_uhcpc
+```
```

Comment:
Actually, do we need this in the readme at this point?
It seems that this check is superfluous in this example at least.
Should we move this to another part of the readme, e.g. in the manual config?